pandas/tseries/tests/test_resample.py

```diff
@@ -572,12 +573,18 @@ def test_agg_consistency(self):
         result = r.agg({'r1': 'mean', 'r2': 'sum'})
         assert_frame_equal(result, expected)
 
+    def test_raises_on_non_datetimelike_index(self):
```

Comment:
done. I had thought generic was to run tests across a number of types, while this doesn't depend on the index type.